.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,37 @@
+---
+name: Continuous Integration
+
+on:
+  push:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+      - name: Install dependencies
+        run: |
+          sudo apt-get install -y libgdal-dev liblwgeom-dev
+          bundle install --jobs 4 --retry 3
+      - name: Run tests
+        run: bundle exec rake spec
```

If you add rspec-github as a dev dependency you can
```suggestion
        run: bundle exec rspec --format RSpec::Github::Formatter --format progress
```